src/ampycloud/prms/ampycloud_default_prms.yml

```diff
@@ -38,6 +38,11 @@ BASE_LVL_HEIGHT_PERC: 5
 # Set to 100 to use all the points.
 BASE_LVL_LOOKBACK_PERC: 100
 
+# If ceilo Ids are set, for any given layer/ group/ slice, only ceilos with the corresponding ID
+# will be used for base height calculation. If the selection results in an empty layer/ group/
+# slice, we go back to using all ceilo hits and ignore the selection.
```

Comment:
I would add here an typical use-case for this parameter. Users may wonder why that parameters could be useful.

Reply:
Done, along with adding a documentation for changes since v2.0.0